src/platform-includes/getting-started-config/apple.mdx

```diff
@@ -1,4 +1,4 @@
-You should also initialize the SDK as soon as possible, such as in your AppDelegate `application:didFinishLaunchingWithOptions` method:
+We recommend initializing the SDK on the main thread as soon as possible, such as in your AppDelegate `application:didFinishLaunchingWithOptions` method:
```

Comment:
m: Should we clarify that we will access ui related classes when automatic tracing is enabled, and if the SDK is initialized in a background thread UIKit will complain about it?

Reply:
I don't think we should mention that. Even without these features, you should call us on the main thread. Otherwise, you might miss some crashes.